Scripts/StixParser/StixParser_test.py

```diff
@@ -219,3 +219,10 @@ def test_main(self, mocker):
         files_path = "./TestData/stix1/"
         mock_demisto(mocker)
         self._run_on_files(files_path, main, mocker)
+
+
+def test_dict_no_stix(mocker):
+    from StixParser import stix2_to_demisto
+    mock_demisto(mocker)
+    with pytest.raises(SystemExit):
```

Comment:
check if `return_error` is called instead, and verify it was called with `"No STIX2 object could be parsed"`